### ISSUE TITLE
Fearure Request

### ISSUE THREAD

**Issue by MrSaeid007** [MatsuriDayo/nekoray#1044]
Dear Nekoray Develpoer

Please add the Reality in profile/setting/security section
<img width="623" alt="Nekoray windows" src="https://github.com/MatsuriDayo/nekoray/assets/69323214/02c9afd9-9a72-49a3-9b2f-ea7d228a926e">
As you can see in the picture, there is no option called Reality and we only see tls
(Transport layer encryption)

Thank you.

**Comment by sawertEND** [comment]
tls= Reality

**Comment by MrSaeid007** [comment]
> tls= Reality

No, it's not because the Nekoray App sometimes can not recognize Reality.

this is the log:
ERROR[0001] dns: exchange failed for [www.msftconnecttest.com](http://www.msftconnecttest.com/). IN A: reality verification failed

If Nekoray develpoer add Reakity it will be great.
